src/main/java/io/github/darealturtywurty/ancientology/client/util/ClientEvents.java

```diff
@@ -27,7 +31,7 @@ private ModEvents() {
 
         @SubscribeEvent
         public static void clientSetup(FMLClientSetupEvent event) {
-
+            BlockEntityRenderers.register(BlockEntityInit.JUMPRASHER.get(), JumprasherBER::new);
```

Comment:
This should use the `RegisterRenderers` event!

Reply:
Oh, didn't know that was a thing.